book/website/communication/open/embedding-ds-ai-disciplines.md

```diff
@@ -0,0 +1,142 @@
+
+1. **Assessment for Learning (AfL)** is used during learning, and it is useufl to identify students' demographic, students’ needs and starting point, and to generate feedback they can use to improve performance. AfL can take different forms: it could be as simple as observing class discussions, asking questions in oral or written form, or using collaborative tools such as Miro or Notion to leverage visual aids and conceptual mapping. 
+
+AfL informs changes you can make to your lesson straight away to make it more effective. Through AfL, students will:
+
+* **Introduce themselves**: understanding *WHO*. General information will help you understand the overall cohort's demographics and map out discipline-specific interests.
+* **Explain what they can do**: understanding *HOW*. Assess existing skills and expertise (this is also helpful for students to understand their capabilities).
+* **Express preferences about what they want to do**: understanding *WHAT*. Let students describe their ideal scenarios; this will help you refine and tailor your content.
+* **Share their goals and aspirations**: understanding *WHY*. What do they need this for?
+
+
+In *Seven Myths of Education* (2014), Daisy Christodoulou suggests that teachers should act as "thermostats, not thermometers" meaning they should not only measure where a student is but also make necessary adjustments to guide them to where they need to be. This perspective is fundamental when thinking about AfL, and to understand the critical role of Effective Feedback.
+
+**Effective feedback in AfL**: Effective feedback requires active listening from both the educator and the student. As part of the questioning process, it is an essential tool for developing students' thinking. Feedback must be task-focused, timely, specific, clear and unbiased. In this way, you will provide your students with information about their current performance and guidance on how they can improve to reach their goals.
+
+2. **Formative Assessment (FA)** supports teaching by assessing a learner's state and inferring next steps (Zhai et al., 2021). It is similar to AfL, as both methods are used to understand student progress and inform teaching. However, while AfL is carried out during learning to inform teaching and identify areas for improvement, FA is used for day-to-day assessments to gauge and explore students' understanding of a topic.
+
+The FA process usually consists of the following three practices (Stanja et al., 2023):
```

Comment:
```suggestion
The formative assessment process usually consists of the following three practices (Stanja et al., 2023):
```

Reply:
I would write these abbreviations out in full to make it easier for people to follow